kasa/smartdevice.py

```diff
@@ -241,7 +248,9 @@ def _verify_emeter(self) -> None:
         if not self.has_emeter:
             raise SmartDeviceException("Device has no emeter")
         if self.emeter_type not in self._last_update:
-            raise SmartDeviceException("update() required prior accessing emeter")
+            raise SmartDeviceException(
+                f"update() required prior accessing emeter: {self._last_update}"
```

Comment:
I'm not sure if we should add the whole payload into the exception message.

Reply:
I agree. I forgot to take it out when I was debugging

👍

Reply:
reverted in 0963aeb275d545f8a88cdf06b6b668579d582091